core/Updates/1.5-b1.php

```diff
@@ -29,7 +32,12 @@ public function __construct(MigrationFactory $factory)
 
     public function getMigrations(Updater $updater)
     {
-        return array(
+        $tableMetadata = new TableMetadata();
```

Comment:
are below changes needed? We could otherwise just add them and remove them later again should someone update from an old version?

Reply:
this was for the build I think

Reply:
If possible, be great to undo the changes and see if it still works. It's not really expected that a change is needed here.Technically, the updater should basically first add these columns as part of the 1.5 update, then remove them later as part of the 4.x update.

Reply:
I see, here's the error: https://travis-ci.org/github/matomo-org/matomo/builds/690764432

there's a test that checks that every update works if we run it again even w/ the latest changes present (`testUpdateWorksAfterPiwikIsAlreadyUpToDate()`). Since we drop the old columns, it fails.